test/formats/test_vorbis.py

```diff
@@ -189,6 +189,18 @@ def test_delete_invalid_tagname(self):
                 del metadata[invalid_tag]
                 save_metadata(self.filename, metadata)
 
+        @skipUnlessTestfile
+        def test_load_strip_trailing_null_char(self):
+            # Create a test file that contains only disctotal / tracktotal,
+            # but not totaldiscs and totaltracks
```

Comment:
This comment looks unrelated to this change, I guess it was copied from `test_delete_totaldiscs_totaltracks()`

Reply:
Yes, this was a copy and paste left-over. Removed